src/Microsoft.AspNet.Razor.Runtime/Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -224,18 +228,24 @@ public void AddTagHelperAttribute(string name, object value)
         }
 
         /// <summary>
-        /// Execute and retrieve the rendered child content asynchronously.
+        /// Execute children asynchronously with the given <paramref name="encoder"/> in scope and return their
+        /// rendered content.
         /// </summary>
+        /// <param name="useCachedResult">If <c>true</c> multiple calls to this method will not cause re-execution
+        /// of child content; cached content will be returned.</param>
+        /// <param name="encoder">
+        /// The <see cref="HtmlEncoder"/> to use. Does not override all HTML encodings which may occur.
```

Comment:
> What other HTML encodings?

1. Another `RazorView`'s or `RazorPage`'s content
2. A view component's content
3. URLs `UrlResolutionTagHelper` and its subclasses touch
4. `@Html.BeginForm()`'s content
5. Anything written out using `FormContext.EndOfFormContent`
6. ...

Do you want the above list in these comments?